playbooks/move_vm_from_on_prem_to_gcp/README.md

```diff
@@ -0,0 +1,74 @@
+# cloud.gcp_ops.move_vm_from_on_prem_to_gcp
+
+A playbook to migrate an existing on prem VM running on KVM hypervisor to GCP.
+
+## Requirements
+
+**qemu** and **qemu-img** packages installed on the host running the KVM hypervisor.
+
+The ``cloud.gcp_ops.clone_one_prem_vm`` requires privilege escalation because the .qcow2 file created by ``virt-clone`` is owned by root and ``qemu-img convert`` requires access to convert it to .raw.
+
+VM Import requires a role to perform certain operations on your behalf. You must create a service role named vmimport with a trust relationship policy document that allows VM Import to assume the role, and you must attach an IAM policy to the role.
+
+GCP credentials with permissions:
+  * to upload an object into a GCP storage bucket;
+  * to create a GCP custom image and get information about it;
+  * to start a GCP compute instance.
+
+## Playbook Variables
+
+* **kvm_host** (dict): Information about the host running the KVM hypervisor that are dynamically added to the inventory.
+    * **name** (str): This is a user-defined name for the host you are adding to the inventory.
+    * **ansible_host** (str): This variable specifies the hostname or IP address of the host you are adding to the inventory.
+    * **ansible_user** (str): This variable specifies the SSH username that Ansible should use when connecting to the host.
+    * **ansible_ssh_private_key_file** (str): This variable specifies the path to the SSH private key file that Ansible should use for authentication when connecting to the host.
+    * **groups**: (list) This variable enabled you to assign the newly added host to one or more groups in the inventory.
+
+### For google.cloud.clone_on_prem_vm
```

Comment:
This is duplicated information, you should add a reference to a the file `roles/clone_on_prem_vm/README.md` here

Reply:
Restructured README to link the roles documentation rather than showing duplicated information. Thanks.